calico/getting-started/kubernetes/windows-calico/quickstart.mdx

```diff
@@ -109,7 +109,7 @@ The following steps install a Kubernetes cluster on a single Windows node with a
    Invoke-WebRequest {{filesUrl}}/scripts/install-calico-windows.ps1 -OutFile c:\install-calico-windows.ps1
    ```
 
-1. Install {{prodnameWindows}} for your datastore with using the default parameters or [customize installation parameters]. (#configure-installation-parameters).
+1. Install {{prodnameWindows}} for your datastore with using the default parameters or [customize installation parameters](#configure-installation-parameters).
```

Comment:
A quick search of the docs shows that this problem occurs 16 times. Can you fix this globally? 
<img width="1168" alt="image" src="https://user-images.githubusercontent.com/39195715/214264686-4d872a29-c01c-4327-9bb0-901c48f34cbd.png">